specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -975,6 +1028,120 @@
     }
   },
   "definitions": {
+    "KeyReleaseCondition": {
+      "properties": {
+        "claim": {
+          "x-ms-client-name": "claim name",
+          "type": "string",
+          "minLength": 1,
+          "description": "claim name"
+        },
+        "condition": {
+          "x-ms-client-name": "condition to test",
+          "type": "string",
+          "minLength": 1,
+          "description": "condition to test",
+          "enum": [
+            "equals"
+          ],
+          "x-ms-enum": {
+            "name": "KeyReleaseCondition",
+            "modelAsString": true,
```

Comment:
Do you really want this to be `true` right now? If you only support "equals" and this dictates how you compare info, it seems this shouldn't be extensible. What if someone passes "foo"?

Reply:
It will be extended in the future, perhaps before GA. If you pass an unrecognised string then the service will return HTTP 400. How do you think this should be encoded?

Reply:
For the various algorithms below (understood, thanks), it makes sense. Just seemed like this was more constrained to boolean operators. Yes, a 40x would make sense here, but a couple things to keep in mind:

1. All our generators for languages treat a single-valued enum as a constant, so as written some of our languages actually won't have an enum. That is probably something we'll have to handle manually (post-generated code).
2. The enums (when we generate them) provide hints to the user to make the API easier to use. If this one in particular might support more before GA, will that make it into swagger so we can regenerate?